app/views/appointment_notif/initial_message.txt.erb

```diff
@@ -0,0 +1,5 @@
+Paired
+<%= MessageGenerator.new.pairing_notification(@pairing.pairee_name, @pairing.date, @pairing.time) %>
```

Comment:
I'd like to abstract this a bit more; can we call MessageGenerator in the updatePairing resolver and pass it to the email service as an argument? That way we're matching how we designed the SmsService. Thanks!